sites/dotnet-guide/contents/tutorials/basics/ado-net/index.md

```diff
@@ -0,0 +1,104 @@
+---
+type: TutorialStep
+date: 2020-06-05
+title: Basics of ADO.NET
+technologies: [.net]
+products: [rider]
+topics: [data]
+author: khalidabuhakmeh
+subtitle: Working with databases using DbConnection, DbCommand, and DbDataReader
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+#   url: TODO: Upload video    
+---
+
+ADO.NET is a data service for .NET developers. In the older version of the .NET framework, Microsoft packaged it along with other libraries. More recently, the ADO.NET libraries ship in NuGet packages, with `System.Data.SqlClient` being specifically for SQL Server access. ADO.NET is a low-level interface to our data storage of choice, and its use depends heavily on our understanding of SQL.
+
+ADO.NET has three commonly used objects that developers will deal with: `DbConnection`, `DbCommand`, and `DbDataReader`. The SQL Server implementations of these classes are `SqlConnection`, `SqlCommand`, and `SqlDataReader`. 
```

Comment:
Every paragraph starts with ADO.NET. Add some variety? `There are three commonly used.... in ADO.NET: ....`